stream_alert/rule_processor/threat_intel.py

```diff
@@ -164,6 +166,10 @@ def _extract_ioc_from_record(self, record):
                         for original_key in original_keys:
                             value = value[original_key]
                     if value:
+                        # Threat Intel will only check against public IP addresses
+                        # when process IP IOCs.
```

Comment:
processing*

Reply:
# Private IP addresses (ex: RFC1918) will not be compared against threat intelligence